kalite/distributed/management/commands/sphinx_docs_preprocess.py

```diff
@@ -0,0 +1,26 @@
+import os
+import re
+
+from django.core.management.base import NoArgsCommand
+
+from kalite.settings import USER_DATA_ROOT
```

Comment:
Can you move this import inside `handle`? This will mess up the central server imports, as this will force all settings to load.